vnext/Microsoft.ReactNative/Utils/ImageUtils.h

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#pragma once
+#include <cxxreact/JSBigString.h>
+#include <future>
```

Comment:
nit: Are those headers needed to understand the decls in this file?

Reply:
Removed